github/github.go

```diff
@@ -154,6 +159,17 @@ type UploadOptions struct {
 	Name string `url:"name,omitempty"`
 }
 
+type MediaRawType int
```

Comment:
This (and a few more below) is an exported package-level symbol. Go style suggests:

> All top-level, exported names should have doc comments, as should non-trivial unexported type or function declarations.

Source: https://github.com/golang/go/wiki/CodeReviewComments#doc-comments